doc/source/blog_posts/TRACER.ipynb

```diff
@@ -0,0 +1,2267 @@
+{
```

Comment:
<p>Can you shorten the post? Maybe "Blending Observations from TRACER using Py-ART". Also, Can you add a link to the TRACER field campaign website, and a bit of an overview of the objective of the field campaign to put this into context/motivation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I have put in a couple of extra sentences about TRACER here and shortened the post to just one case as well as the title. In addition to the brief introduction I've added a link to the TRACER web page.

Reply:
Yay! Thanks :)